infra/terraform/external_function/snowalert.tf

```diff
@@ -25,14 +34,48 @@ variable "snowflake_integration_user" {
   default     = null
 }
 
-variable "snowflake_integration_external_id" {
-  type        = string
-  description = "API_AWS_EXTERNAL_ID from DESC INTEGRATION ..."
-  default     = "000000"
-}
-
 variable "aws_cloudwatch_metric_namespace" {
   type        = string
   description = "where EF can write CloudWatch Metrics"
   default     = "*"
 }
+
+variable "gateway_logger_name" {
+  type        = string
+  default     = "gateway_logger1"
+}
+
+variable "gateway_caller_name" {
+  type        = string
+  default     = "gateway_caller1"
+}
+
+variable "aws_iam_role_policy_name" {
+  type        = string
+  default     = "gateway_policy1"
+}
+
+variable "aws_deployment_stage_name" {
+  type        = string
+  default     = "prod"
+}
+
+variable "snowflake_username" {
+  type        = string
+  default     = ""
+}
+
+variable "snowflake_account" {
+  type        = string
+  default     = ""
+}
+
+variable "snowflake_password" {
+  type        = string
+  default     = ""
+}
+
+variable "snowflake_role" {
+  type        = string
+  default     = ""

```

Comment:
it would help to make the name more specific, or give it a description -- e.g. this is the role that will own the API Integration?